app/views/GuidanceView.scala.html

```diff
@@ -1,180 +1,228 @@
-        Start now
-        <svg class="govuk-button__start-icon" xmlns="http://www.w3.org/2000/svg" width="17.5" height="19" viewBox="0 0 33 40" aria-hidden="true" focusable="false">
-            <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z" />
-        </svg></a>
+        @bullets(Seq("your EORI number (If you do not have an EORI number we will direct you to the service to get one.)",
+            "details that identify you or your business so that we can verify your use of the EORI number",
+            "to be an administrator user to set up the subscription for an organisation."
+        ).map(Html(_)))
 
-    @h2("After you have disclosed", Some("5"))
+        <div class="govuk-inset-text">
+            You need to complete the disclosure in one go. You cannot save your answers and return later.
+        </div>
 
-    @p(Html("If we agree with the calculations you’ve provided, we’ll send a C18 Post Clearance Demand Note in the post within 14 days."))
+        <a href="@controllers.routes.IndexController.onPageLoad()" role="button" draggable="false" class="govuk-button govuk-button--start" data-module="govuk-button">
+            Start now
+            <svg class="govuk-button__start-icon" xmlns="http://www.w3.org/2000/svg" width="17.5" height="19" viewBox="0 0 33 40" aria-hidden="true" focusable="false">
+                <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z" />
+            </svg></a>
 
-    @p(Html("This will confirm what you owe to HMRC and, if you have not paid by deferment, will tell you how to pay. You must pay within 10 days of receiving the demand note otherwise you’ll be charged interest."))
+        @h2("After you have disclosed", Some("5"))
 
-    @p(Html("If you’ve paid too much import VAT or customs duties apply for repayment of import duties or import VAT."))
+        @p(Html("If we agree with the calculations you’ve provided, we’ll send a C18 Post Clearance Demand Note in the post within 14 days."))
 
-    @p(
-        Html(
-            <span>
-                If you have any questions or have not received the demand note within 14 days, email
-                <a class="govuk-link" href="mailto:customsaccountingrepayments@hmrc.gov.uk">customsaccountingrepayments@hmrc.gov.uk</a>.
-            </span>.toString()
+        @p(Html("This will confirm what you owe to HMRC and, if you have not paid by deferment, will tell you how to pay. Y ou must pay within 10 days of receiving the demand note otherwise you’ll be charged interest."))
+
+        @p(Html("If you’ve paid too much import VAT or customs duties apply for repayment of import duties or import VAT."))
+
+        @p(
+            Html(
+                    <span>
+                    If you have any questions or have not received the demand note within 14 days, email
+                    <a class="govuk-link" href="mailto:customsaccountingrepayments@hmrc.gov.uk">customsaccountingrepayments@hmrc.gov.uk</a>
+                    .
```

Comment:
If you place the dot on a new line like this, the rendered HTML will have padding between the text and the dot. Not sure if it's a big issue, but it gets rendered differently.